### PR TITLE
fix GetLinkModal link prop

### PR DESCRIPTION
**Pull request by dorakemon**
## What changes are proposed in this pull request?

Fixed for #3583 #5195 

When I create web app on Chrome and embeded mlflow ui in my app as iframe content,

My app: `localhost:3000`
Mlflow ui: `localhost:5000`
(different origin) 

I encountered cors Policy error, because mlflow ui cannot access myapp `window.top`.
Mlflow ui does not create new window, so `window.location.href` works correctly as the same as before.

## How is this patch tested?

Load mlflow ui in iframe and mlflow ui is successfully loaded.

## Does this PR change the documentation?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly by following the steps below.

1. Check the status of the `ci/circleci: build_doc` check. If it's successful, proceed to the
   next step, otherwise fix it.
2. Click `Details` on the right to open the job page of CircleCI.
3. Click the `Artifacts` tab.
4. Click `docs/build/html/index.html`.
5. Find the changed pages / sections and make sure they render correctly.

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [x] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [x] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by dbczumar — APPROVED** [review]
LGTM! 

cc @sunishsheth2009 @harupy Can you TAL as well?

File: mlflow/server/js/src/experiment-tracking/components/ExperimentView.js [hunk 1]
```diff
@@ -400,7 +400,7 @@ export class ExperimentView extends Component {
     const { showGetLinkModal } = this.state;
     return (
       <GetLinkModal
-        link={window.top.location.href}
+        link={window.location.href}
```
[the remark this answers is not in the document]
Comment (dorakemon) [reply mlflow/server/js/src/experiment-tracking/components/ExperimentView.js:403]: @harupy cc @sunishsheth2009 
Thank you for fast reviewing.

I don't know Dynamics well, but is below link related to the issue?

https://docs.microsoft.com/en-us/power-apps/developer/model-driven-apps/best-practices/business-logic/avoid-window-top

[the remark this answers is not in the document]
Comment (harupy) [reply mlflow/server/js/src/experiment-tracking/components/ExperimentView.js:403]: Hi @dorakemon, can we fix the code like below?

```js
  getUrl() {
    return window.location.href;
  }

  renderGetLinkModal() {
    const { showGetLinkModal } = this.state;
    return (
      <GetLinkModal
        link={this.getUrl()}
        ...
```

This is for parity with our private mlflow UI repository in Databricks.

File: mlflow/server/js/src/experiment-tracking/components/ExperimentView.js [hunk 2]
```diff
@@ -396,11 +396,15 @@ export class ExperimentView extends Component {
   }
   // END-EDGE
 
+  getUrl() {
```
Comment (dorakemon) [inline mlflow/server/js/src/experiment-tracking/components/ExperimentView.js:399]: @harupy 
This works well even if in iframe.
I commit small change but if you've already fixed, please close this pull request.
Thank you for teaching and maintaining very good mlflow UI.

Comment (harupy) [reply mlflow/server/js/src/experiment-tracking/components/ExperimentView.js:399]: @dorakemon 

> I commit small change but if you've already fixed, please close this pull request.

We've fixed the issue in our private repository and it'll be shipped in the next MLflow release. Again, thank you for finding this issue. We really appreciate it.

**Pull request closed without merging**